catalog/spatial/geocoding/spatial-geocoding-extract/src/main/java/org/codice/ddf/spatial/geocoding/extract/GeoNamesFileExtractor.java

```diff
@@ -161,7 +176,7 @@ private InputStream getInputStreamFromResource(
       } else if (resource.matches("((?i)cities[0-9]+)")) {
         resource = resource.toLowerCase();
```

Comment:
❓I know that these aren't your changes, but why do we need to modify the case here for country codes and cities? Doesn't it just make a URL connection and URLs are case-insensitive?

Reply:
The filename part of the URL is case sensitive.

Reply:
Oh, good point. I just confirmed that `http://download.geonames.org/export/dump/aU.zip` doesn't work.